adminSiteClient/GdocsIndexPage.tsx

```diff
@@ -1,155 +1,281 @@
+                                ) : null}
+                                <Link
+                                    to={`${this.props.match.path}/${gdoc.id}/preview`}
+                                >
+                                    <h5 className="gdoc-index-item__title">
+                                        {gdoc.content.title}
+                                    </h5>
+                                </Link>
+                                <a
+                                    href={`https://docs.google.com/document/d/${gdoc.id}/edit`}
+                                    className="gdoc-index-item__edit-link"
+                                >
+                                    Edit
+                                </a>
+                                <p className="gdoc-index-item__byline">
+                                    {gdoc.content.byline}
+                                </p>
+                                <span className="gdoc-index-item__tags">
+                                    {gdoc.content.type !==
+                                        OwidGdocType.Fragment && gdoc.tags ? (
+                                        <EditableTags
+                                            tags={gdoc.tags}
+                                            onSave={(tags) =>
+                                                this.context?.updateTags(
+                                                    gdoc,
+                                                    tags as any
+                                                )
+                                            }
+                                            suggestions={this.tags}
+                                        />
+                                    ) : null}
+                                </span>
+                            </div>
+                            <div className="gdoc-index-item__publish-status">
+                                {gdoc.published ? (
+                                    <a
+                                        title={
+                                            gdoc.publishedAt
+                                                ? new Date(
+                                                      gdoc.publishedAt
+                                                  ).toDateString()
+                                                : undefined
+                                        }
+                                        href={
+                                            gdoc.content.type !==
+                                            OwidGdocType.Fragment
+                                                ? `${BAKED_BASE_URL}/${gdoc.slug}`
+                                                : undefined
+                                        }
+                                        className="gdoc-index-item__publish-link"
+                                    >
+                                        Published
```

Comment:
the green "published" linking out to the live version is a nice touch, but is somewhat misleading. Its button appearance makes it look like an action is being carried out upon click (unpublishing?). Maybe "Live ->" would work better ("->" being the icon with the arrow pointing out of box), with the link opening in a new tab?